New fuctional - run in terminal with parametrs.

LevPasha opened the pull request:
I think you are the master of this package and should make modal window - on your own. ;)

LevPasha committed:
New fuctional - run in terminal with parametrs.

I think you are the master of this package and should make modal window - on your own. ;)

pohmelie commented on lib/run-in-terminal.coffee:
```diff
@@ -230,7 +248,10 @@ module.exports =
         atom.contextMenu.add(menu)
 
-    tree_start_terminal_here: (run) ->
+    tree_start_terminal_here: (run, par) ->
```
`par` should be replaced with `ask_parameters` here and below, since there is no reason to make short unreadable names.

pohmelie commented on lib/run-in-terminal.coffee:
```diff
@@ -180,37 +180,55 @@ module.exports =
+            [
+                selectors.tree,
+                "tree-start-terminal-here-and-run-with-par",
+                "Start terminal here and run [tree] (with par)",
```
`with par` here and below should be replaced with `with parameters`, since this is what user see and this should definetly be maximum readable.

LevPasha replied:
![default](https://cloud.githubusercontent.com/assets/9821835/12744143/99f30c58-c9a3-11e5-8717-a81641366570.png)

I think you use very long names. Maybe you should change it to `Start terminal`, `Run in terminal` and `Run in terminal with parameters`. Maybe this name will have some sorting in menu problem? But `[tree]`, `[editor]` and etc - this is trash )))

pohmelie replied:
@LevPasha 
I don't use context menu, but it looks like there will be no problems with sorting, since there is no sorting :smile: 
About naming. Maybe you are right… I think this can be toggled with something like «use short context names» option.

LevPasha committed:
New fuctional - run in terminal with parametrs.

pohmelie added a note 11 hours ago
par should be replaced with ask_parameters here and below, since there is no reason to make short unreadable names.

pohmelie commented:
Implemented in https://github.com/pohmelie/run-in-terminal/commit/92f1607cfc22f520d8f901d8a0b63dfeb91bf994 :wink:
